pyincore_viz/geoutil.py

```diff
@@ -1140,6 +1140,78 @@ def get_ipyleaflet_heatmap(locations=None, radius=10, blur=10, max=1, name=""):
 
         return heatmap
 
+    @staticmethod
+    def plot_local_earthquake(eq_dataset):
+        """
+        Plot local earthquake data on the map
+        """
+        demand_type = eq_dataset.demand_type
```

Comment:
could we also include demand_units as well?

Reply:
Also hazard_type, to be consistent

Reply:
added those